crates/eth-rpc/src/servers/kakarot_custom_rpc.rs

```diff
@@ -0,0 +1,27 @@
+use jsonrpsee::core::{async_trait, RpcResult as Result};
+use kakarot_rpc_core::client::api::KakarotEthApi;
+use kakarot_rpc_core::models::balance::TokenBalances;
+use reth_primitives::Address;
+use starknet::providers::Provider;
+
+use crate::api::kakarot_custom_api::KakarotCustomApiServer;
+
+/// The RPC module for the Ethereum protocol required by Kakarot.
+pub struct KakarotCustomRpc<P: Provider + Send + Sync> {
```

Comment:
I think I got this API from alchemy
https://docs.alchemy.com/reference/ethereum-api-endpoints

let's rename it to give them many thanks

Reply:
Makes sense. Resolved.

Also, then it means we are not completely implementing the Token Api, we just have one method, should I make an issue for this?

<img width="799" alt="image" src="https://github.com/kkrt-labs/kakarot-rpc/assets/41180869/ae43b217-7525-4426-b9fb-4e34bf681aa6">

Reply:
imho these `alchemy_` endpoints are very low priority for us. It is was good catch to realize that we didn't serve what we thought we were serving, and we definitely could define one trait per prefix even in the "standard" et json rpc spec (`eth_`, `net_`, `web3_`). This `alchemy_` RPC is very low priority for us, so why not an issue, but with "nice to have", and "low priority" tags!

Reply:
@ClementWalter , noted, whenI make the issue, I will make sure to use these tags.

Reply:
It has been renamed accordingly -> 
<img width="1019" alt="image" src="https://github.com/kkrt-labs/kakarot-rpc/assets/41180869/ae9f9c15-5f92-49d4-9766-1e716534ac25">